### PR TITLE
resource_contorl: add privilege requirement for dashboard resouce control page

### PR DESCRIPTION
**Pull request by glorv**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-statements/sql-statement-calibrate-resource.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ TiDB 提供两种预估方式：
 
 - [`tidb_enable_resource_control`](/system-variables.md#tidb_enable_resource_control-从-v660-版本开始引入) 需要处于开启状态
 - 需要拥有 `SUPER` 或者 `RESOURCE_GROUP_ADMIN` 权限
-- 需要拥有 `METRICS_SCHEMA` 库下所有表的 `SELECT` 权限
+- [根据实际负载估算容量](#根据实际负载估算容量) 功能需要拥有 `METRICS_SCHEMA` 库下所有表的 `SELECT` 权限
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-calibrate-resource.md:58]:
```suggestion
- [根据实际负载估算容量](#根据实际负载估算容量)功能需要拥有 `METRICS_SCHEMA` 库下所有表的 `SELECT` 权限
```

File: dashboard/dashboard-resource-manager.md [hunk 2]
```diff
@@ -63,6 +63,10 @@ summary: 介绍如何使用 TiDB Dashboard 的资源管控页面查看资源管
 
   可以通过[监控指标](#监控指标)中的 **CPU Usage** 选择合适的时间范围。
 
+> **注意：**
+>
+> 使用`容量估算`功能需要当前登录用户具有 `SUPER` 或 `RESOURCE_GROUP_ADMIN` 权限以及部分系统表的访问权限，在使用此功能前，请确保当前用户已具备相关的权限，否则部分功能将无法正确使用。详细请参考 [容量预估的权限要求](/sql-statements/sql-statement-calibrate-resource.md#权限)
```
Comment (Oreoxmt) [inline dashboard/dashboard-resource-manager.md:68]:
```suggestion
> 要使用容量估算功能，当前登录用户需要拥有 `SUPER` 或 `RESOURCE_GROUP_ADMIN` 权限，并拥有部分系统表的访问权限。在使用此功能前，请确保当前用户已拥有这些权限，否则部分功能可能无法正常使用。详情请参考[容量预估的权限要求](/sql-statements/sql-statement-calibrate-resource.md#权限)。
```

**Review by Oreoxmt — APPROVED** [review]

**Comment by glorv** [comment]
@nolouch PTAL

**Comment by glorv** [comment]
@nolouch  PTAL

**Review by nolouch — APPROVED** [review]

**Pull request merged**